src/OpenTelemetry.OpAmp.Client/OpenTelemetry.OpAmp.Client.csproj

```diff
@@ -13,4 +13,32 @@
     <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Google.Protobuf" Version="3.31.1" />
+    <PackageReference Include="Grpc.Tools" Version="2.72.0">
```

Comment:
Just a note: It will not work on ARM, if anybody decide to compile on this machine.

https://github.com/grpc/grpc/issues/38538